src/cmd-kola

```diff
@@ -80,6 +80,10 @@ kolaargs.extend(['--output-dir', outputdir])
 kolaargs.extend(args.subargs or [default_cmd])
 kolaargs.extend(unknown_args)
 
+# FIXME drain into kola once it understands cosa
+if os.path.isdir('src/config/tests/kola'):
+    kolaargs.extend(['-E', 'src/config'])
```

Comment:
Hmm I think we need to make this conditional on whether `args.subargs[0]` is `run` or we defaulted to `run` so that we don't pass it to e.g. `kola run-upgrade` which would error out.

But yeah, all this will be cleaner once `cmd-kola` just disappears.